frontend/public/components/utils/log-window.jsx

```diff
@@ -131,7 +131,9 @@ class LogWindowWithTranslation extends React.PureComponent {
         <div className="log-window__body">
           <div className="log-window__scroll-pane" ref={this._setScrollPane}>
             <div className="log-window__contents" ref={this._setLogContents} style={{ height }}>
-              <div className="log-window__lines">{content}</div>
+              <div className={`log-window__lines${this.props.wrapLines ? '--wrap' : ''}`}>
```

Comment:
Add `import classnames from 'classnames';` in the imports at the top of the file.
Add `wrapLines` on line 117 since we are already destructuring.
Then:
```suggestion
              <div className={classnames('log-window__lines', { 'log-window__lines--wrap': wrapLines })}>
```

Reply:
Made suggested changes. Resolving conversation